book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
```

Comment:
- [ ] **Use proper naming conventions**: Some variable and function names could be more descriptive to enhance code readability.  For instance, I suggest renaming the `editToDo` function to `handleEditTodo` to provide a clearer indication of its purpose and functionality.